doc/pages/example_notebooks/Diffusion/DiffusionValidationI.v4.ipynb

```diff
@@ -27,14 +27,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%matplotlib notebook\n",
+    "%matplotlib inline\n",
     "\n",
     "import pandas as pd\n",
     "import matplotlib.pyplot as plt\n",
     "import seaborn as sns\n",
     "import numpy as np\n",
     "from scipy.stats import chisquare\n",
     "from scipy.integrate import quad\n",
+    "import sys\n",
+    "sys.path.append('../../../../build/')\n",
```

Comment:
Here again, why is this path appended? Seems like bad practice for a general example.

Reply:
Fixed.